blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -221,6 +215,16 @@ public void trackResponse(HttpRequest request, HttpResponse response) {
 
     }
 
+    public void trackDelayedMetricsTenant(String tenantid, List<Metric> delayedMetrics) {
+        if (isTrackingDelayedMetrics) {
+            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metrics %s\n",tenantid);
+            for (Metric metric : delayedMetrics) {
+                logMessage += String.format("%s has collectionTime %d\n", metric.getLocator().toString(), metric.getCollectionTime());
```

Comment:
Ok thinking it through, I realize it's not hard to just use the current system time and calculate the difference between that and collectionTime and log that also, since that's what we'll be doing anyway using the log timestamp.  I hope this is good enough for now.  

The more *correct* way though is to add a new "creationTime" for each metric (add a new property in the Metric class and timestamp it in the constructor), then subtract that from collectionTime to calculate the actual delays.  But doing so will require us changing all the serialization code, which is too intrusive just for this change.  (If we do have a reason to track the ingestion/class creation time in the fgure, however, then the code can be modified to use that).